crates/cxx-qt-gen/src/generator/naming/invokable.rs

```diff
@@ -23,20 +23,12 @@ impl From<&ImplItemMethod> for QInvokableName {
     fn from(method: &ImplItemMethod) -> Self {
         let ident = &method.sig.ident;
         Self {
-            name: name_from_ident(ident),
+            name: CombinedIdent::from_rust_function(ident.clone()),
```

Comment:
we seem to have a style difference here.

Before we had module + function name -> CombinedIdent. Whereas this one is using a from method on the struct. We should ensure these are consistent.

At the moment they were split so that each of the naming implementations are in their own module so they are easy to find and we don't end up with a huge file again.

Maybe have another commit that does the `from_nnn` route but puts the implementation of each of them in the module files so that they can be found easily?

Eg have `from_rust_function` in a methods file. And keep the others in their own files but change the `nnn_from_ident() -> CombinedIdent` to become `CombinedIdent::from_nnn`. Although some of these are private methods currently and i wonder if they would be optimised to being inlined currently.
```rust
impl CombinedIdent {
  pub fn from_nnn(T) -> Self ...
}
```

Reply:
I've moved this onto CombinedIdent now.
Whilst this results into somewhat longer method names, I think it's more expressive, as it's clearer whats constructed, and also clearer what the input "ident" should be.
Those changes are in a separate commit, so would be easy to undo if you prefer the previous style.

Reply:
I like this way so far, as it keeps the implementations in separate files but still is rusty with T::from_nnn(...)